api/app/utils.py

```diff
@@ -10,12 +12,26 @@ def create_hash():
 
 def get_version_info() -> dict:
     """Reads the version info baked into src folder of the docker container"""
+    release_please_manifest_location = "./.release-please-manifest.json"
+    manifest_versions = None
+
+    if os.path.isfile(release_please_manifest_location):
+        manifest_versions = json.loads(
+            _get_file_contents(release_please_manifest_location)
```

Comment:
We're doing the check twice here since `_get_file_contents` has a `try` / `except` for `FileNotFoundError` which results in `get_file_contents` returning `"unknown"`. With this approach, the path in `_get_file_contents` is redundant.

Reply:
Tweaked this